tests/python/conftest.py

```diff
@@ -10,7 +10,7 @@
 else:
     print("NO mpi4py load")
 
-@pytest.fixture(autouse=True, scope='session')
+@pytest.fixture(autouse=True, scope='function')
```

Comment:
Looks like I have to work more on these scopes:
```
../../../tests/python/test_impactx.py::test_impactx_nofile ERROR
../../../tests/python/test_impactx.py::test_impactx_noparticles amrex::Error::0::ParmParse::Initialize(): already initialized! !!!
SIGABRT
```

Reply:
I needed to change to `session` because I want to reset the `ParmParse` parameters, so that things like `algo.particle_shape` are truly unset for the next test.

Reply:
Funny, problem does not show up locally.

Reply:
Found it finally :tada: 
https://github.com/AMReX-Codes/amrex/pull/2936